TenancyInformationApi/V1/UseCase/ListTenancies.cs

```diff
@@ -20,13 +20,14 @@ public ListTenancies(ITenancyGateway gateway, IValidatePostcode validatePostcode
         }
 
         public ListTenanciesResponse Execute(int limit, int cursor, string addressQuery, string postcodeQuery,
-            bool leaseholdsOnly, bool freeholdsOnly)
+                bool leaseholdsOnly, bool freeholdsOnly, string propertyReference)
         {
-            limit = limit < 10 ? 10 : limit;
-            limit = limit > 1000 ? 1000 : limit;
+            limit = Clamp(limit, 10, 1000);
```

Comment:
Nice!

Reply:
although i think there's a version of this function in .Net framework anyway in the Math class?

Reply:
There probably is but I was impatient :grin: